_posts/2019-03-07-switching-app-frameworks-git-branching.markdown

```diff
@@ -3,6 +3,7 @@ layout: post
 title: "Switching App Frameworks and Using Git Branching"
 date: 2019-03-07 09:30
 categories: upcoming
```

Comment:
Could you change `categories: upcoming` to `categories: meetings`, so the post is moved to the past meetings group?